specification/edgeorder/resource-manager/Microsoft.EdgeOrder/2020-12-01-preview/examples/ListProductFamiliesByResourceGroup.json

```diff
@@ -0,0 +1,326 @@
+                            "disabledReason": "None"
+                          }
+                        },
+                        "name": "AzureStackEdgePro-GPU",
+                        "id": "/subscriptions/fa68082f-8ff7-4a25-95c7-ce9da541242f/resourceGroups/testrg/providers/Microsoft.EdgeOrder/productfamilies/AzureStackEdge/productLines/AzureStackEdge/product/AzureStackEdgePro-GPU",
+                        "type": "Microsoft.EdgeOrder/ProductFamilies/ProductLines/Product"
+                      },
+                      {
+                        "properties": {
+                          "configurations": [],
+                          "displayName": "Azure Stack Edge Pro - FPGA",
+                          "description": {
+                            "descriptionType": "Base",
+                            "shortDescription": "Azure managed physical edge compute device",
+                            "longDescription": "Azure Stack Edge is an AI-enabled edge computing device with network data transfer capabilities. The device is powered with Intel Aria to provide accelerated AI inferencing at the edge. You can choose from the available configurations with one or two FPGAs basis your business need",
+                            "keywords": [
+                              "Edge",
+                              "FPGA"
+                            ],
+                            "attributes": [
+                              "1U device shipped by Microsoft",
+                              "Hardware accelerated ML using Intel ARIA FPGA"
+                            ],
+                            "links": [
+                              {
+                                "linkType": "TermsAndConditions",
+                                "linkUrl": "https://aka.mx/tc"
+                              },
+                              {
+                                "linkType": "KnowMore",
+                                "linkUrl": "https://aka.mx/knowmore"
+                              }
+                            ]
+                          },
+                          "imageInformation": [
+                            {
+                              "imageType": "MainImage",
+                              "imageUrl": "https://xyz.com"
+                            },
+                            {
+                              "imageType": "BulletImage",
+                              "imageUrl": "https://abc.com"
+                            },
+                            {
+                              "imageType": "GenericImage",
+                              "imageUrl": "https://xyz.com/1"
+                            },
+                            {
+                              "imageType": "GenericImage",
+                              "imageUrl": "https://abc.com/1"
+                            }
+                          ],
+                          "costInformation": {
```

Comment:
>costInformation [](start = 27, length = 15)

What is the integration with the Azure COST team (Microsoft.CostManagement)? They also have meters and other information.

Reply:
Yeah we integrate with Billing API's and just return the meter id here